app/assets/stylesheets/_base.scss

```diff
@@ -61,10 +61,12 @@ ul {
   }
 }
 
-
 a, button {
   &:focus {
     outline: 0;
   }
 }
 
+.flex {
```

Comment:
I'm not a really fan of this class... Why do you need a class to set 'display: flex'?